packages/timetable/modulefederation.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   name: "timetable",
   exposes: {
     "./App": "./src/App",
-    "./TimeTableRoute": "./src/pages/TimeTableRoute.js",
+    "./TimeTableRoute": "./src/pages/TimeTableRoute/index.js",
```

Comment:
```
"./TimeTableRoute": "./src/pages/TimeTableRoute",
```

We can simply do this it will automatically import the index file

Reply:
I tried to do that earlier but it did not worked because of that i added the index.js in that path